packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -466,26 +479,7 @@ const renderer =
                 },
                 plugins: [
                     ...config.plugins,
-
-                    // This must only appear on one config – this one is the only mandatory one.
```

Comment:
I don't understand the why. Why does this must appear on on config? I'd imagine it should be fine if we copy the same files to the same location multiple times?

Reply:
Oli wrote this a long time ago, I think it means "at least one" not "must not be more than one"